plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -547,7 +545,7 @@ CacheStats getStatisticsCacheStats()
 
     private static <T, V> void invalidateCache(Cache<T, V> cache, Predicate<T> filterFunction)
```

Comment:
simply pass `EvictableCache` as argument here

Reply:
You mean to remove the cast? 
I want to return the code back to interface-based only, when TODO in `io.trino.plugin.base.cache.EvictableCache#asMap` is resolved.